### PR TITLE
Remove unused variables dom7-utils.js

### PR DESCRIPTION
**Pull request by blikblum**
This PR does:

* Fix typo in comment Utilites > Utilities
* Remove duplicate `url` variable declaration in `parseUrlQuery` (url parameter creates an implicit var)
* Remove unused variables doneLeft and doneTop in `scrollTo`

### CODE REVIEW

**Comment by nolimits4web** [comment]
Thanks!

File: src/js/dom7-utils.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-// DOM Library Utilites
+// DOM Library Utilities
 $.parseUrlQuery= function (url) {
-    var url = url || location.href;
+    url = url || location.href;
```
Comment (ZanderBrown) [inline src/js/dom7-utils.js]: Woah now we are modifying parameters which isn't great.

How about `var toParse = url || location.href;` and continue with that?

Comment (blikblum) [inline src/js/dom7-utils.js]: I am OK with that.

Comment (nolimits4web) [inline src/js/dom7-utils.js]: @ZanderBrown `url` is a string so it won't be modified as it passed by value not by the link

Comment (blikblum) [inline src/js/dom7-utils.js]: Anyway, if appropriate i can prepare a PR with the suggestion

**Pull request merged**